deployments/arbitrum-goerli/usdc/relations.ts

```diff
@@ -8,6 +8,24 @@ export default {
   ClonableBeaconProxy: {
     artifact: 'contracts/ERC20.sol:ERC20'
   },
+  // Native USDC
+  '0xfd064A18f3BF249cf1f87FC203E90D8f650f2d63': {
+    artifact: 'contracts/ERC20.sol:ERC20',
+    delegates: {
+      field: {
+        slot: '0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc'
+      }
+    }
+  },
+  // USDC
+  '0x8FB1E3fC51F3b789dED7557E680551d93Ea9d892': {
```

Comment:
Do we need this still? If so, comment should be `USDC.e`

Reply:
Removed 👍